### DISCUSSION TITLE
测试异常哪里错了吗

### DISCUSSION THREAD

**Discussion by xiaoyao66886** [gkd-kit/gkd#831] [Q&A] [answered]
![Screenshot_2024-12-22-21-00-38-64_3aea4af51f236e4932235fdada7d1643.jpg](https://github.com/user-attachments/assets/399ea3d8-daf5-469f-b255-4038bebc1f5e)

https://i.gkd.li/i/18218403
{
  "key": 2,
  "name": "广告弹窗关闭按钮",
  "rules": [
    {
      "key": 0,
      "fastQuery": true,
      "activityIds": "com.example.app.MainActivity",
      "matches": "[id=\"ad_close_btn\"]",
      "snapshotUrls": "https://i.gkd.li/i/18218403"
    },
    {
      "key": 1,
      "fastQuery": false,
      "activityIds": "com.example.app.MainActivity",
      "anyMatches": [
        "[text=\"关闭\"]",
        "[content-desc=\"关闭\"]"
      ],
      "snapshotUrls": "https://i.gkd.li/i/18218403"
    }
  ]
}

**Comment by lisonge** [comment, marked as answer]
https://github.com/orgs/gkd-kit/discussions/751#discussioncomment-10978281

**Reply by xiaoyao66886** [reply]
谢谢  迷迷糊糊的不懂
